includes/class-swedbank-pay-scheduler.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Swedbank Pay Scheduler Class.
+ * Handles scheduled tasks for Swedbank Pay.
+ *
+ * @package SwedbankPay\Checkout\WooCommerce
+ */
+
+namespace SwedbankPay\Checkout\WooCommerce;
+
+use WC_Logger;
+
+defined( 'ABSPATH' ) || exit;
+
+
+/**
+ * Swedbank_Pay_Scheduler Class.
+ */
+class Swedbank_Pay_Scheduler {
+	public const ACTION_ID = 'swedbank_pay_scheduler_run';
+
+	/**
+	 * @var Swedbank_Pay_Scheduler
+	 */
+	private static $instance = null;
+
+	/**
+	 * Singleton pattern
+	 *
+	 * @return Swedbank_Pay_Scheduler
+	 */
+	public static function get_instance() {
+		if ( null === self::$instance ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	/**
+	 * Logger instance.
+	 *
+	 * @var WC_Logger
+	 */
+	private $logger;
```

Comment:
Chore: Class properties should be declared at the start of the class together. Move this up above the `get_instance` method to follow that convention.